index.html

```diff
@@ -1,450 +1,109 @@
h at least the degree of care that would be exercised by a prudent person given the sensitivity of the personal information. Please continue reading below for more information about our security practices.</p><p>In all of these circumstances, Apptegy is only a data processor. This means we only carry out and process, via our Services, the activities of and instructions of your School (and End Users using the Services under School-Created User Accounts).</p><p><strong>Simply put: when we process activities and instructions from a School that include personal information about you </strong>(<strong>or by you or another School-Created User Account under a School</strong> <strong>account), we are not responsible for the disclosures made or the use of your personal information by the School</strong> <strong>(or School-Created User Account under the</strong> <strong>School</strong> <strong>account). It is your School that has the responsibility to protect your privacy in these instances.</strong></p><p>Please note that the privacy and security policies of your School may differ from our policies. In those cases, our policies will not overrule or change the separate policies of your School, and our policies will not impact or change your School’s separate agreements with and obligations to you. <strong>Instead, the</strong> <strong>privacy and security policies of your School</strong> <strong>govern your personal information and your rights with respect to your School, and not our policies</strong>. As such, you are required to work with your School directly with respect to your personal information. For example, if you request a copy of the personal information we hold about you in order to provide the Services, we will direct you to your School for that information.</p><p>Even though we are not primarily responsible for your personal information in these circumstances, we still want you to understand what personal information we might collect about you as the data processor providing the Services and how we collect it.</p><p>Schools (and End Users under School-Created User Accounts) typically use our Services to publish, upload, manage, store, and/or send communications, messages, announcements, videos, assignments, materials, electronic signature requests, and other content, documents, and information. Thrillshare Media is typically used when the activity is relevant to the School and its End Users generally; Thrillshare Rooms is typically used when the activity is relevant to a particular subset of the School’s End Users – usually a classroom or group at the School.</p><p>However, personal information may be collected for or as a result of any of the activities permitted on the Services. As explained above, Schools (and End Users under School-Created User Accounts) make these choices and are responsible for them.</p><p>Specifically, the following personal information may be collected in connection with the Services:</p><!--kg-card-begin: html--><ul>
```

Comment:
small thing, but we actually host at thrillshare.com and edurooms.com. www.thrillshare.com and www.edurooms.com  are redirected to those root domains